docs/source/_notebooks/keras-text-classifiers.rst

```diff
@@ -0,0 +1,3685 @@
5" title="0.005">wasn&#x27;t</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 95.90%); opacity: 0.81" title="0.001">going</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 95.15%); opacity: 0.81" title="0.001">to</span><span style="opacity: 0.80"> work </span><span style="background-color: hsl(120, 100.00%, 98.55%); opacity: 0.80" title="0.000">out</span><span style="opacity: 0.80"> and his performance </span><span style="background-color: hsl(120, 100.00%, 93.49%); opacity: 0.81" title="0.001">was</span><span style="opacity: 0.80"> quite </span><span style="background-color: hsl(120, 100.00%, 98.36%); opacity: 0.80" title="0.000">&lt;OOV&gt;</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 91.97%); opacity: 0.82" title="0.002">so</span><span style="opacity: 0.80"> all you madison fans </span><span style="background-color: hsl(120, 100.00%, 94.90%); opacity: 0.81" title="0.001">give</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 97.06%); opacity: 0.80" title="0.000">this</span><span style="opacity: 0.80"> a miss </span><span style="background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80" title="0.000">&lt;PAD&gt;</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80" title="0.000">&lt;PAD&gt;</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80" title="0.000">&lt;PAD&gt;</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80" title="0.000">&lt;PAD&gt;</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80" title="0.000">&lt;PAD&gt;</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80" title="0.000">&lt;PAD&gt;</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80" title="0.000">&lt;PAD&gt;</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80" title="0.000">&lt;PAD&gt;</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80" title="0.000">&lt;PAD&gt;</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80" title="0.000">&lt;PAD&gt;</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80" title="0.000">&lt;PAD&gt;</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80" title="0.000">&lt;PAD&gt;</span><span style="opacity: 0.80"> </span><span style="background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80" title="0.000">&lt;PAD&gt;</span
```

Comment:
Looks like we can strip ``pad_value`` both from the start and the end, thus removing ``padding`` argument from the API.